package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
Well, the thing is that the label says `VSCode Icons` and that's what the extension is known for.